linode_api4/objects/serializable.py

```diff
@@ -13,6 +44,16 @@ class JSONObject:
     fields and static typing.
     """
 
+    filters: ClassVar[JSONFilterGroup] = None
```

Comment:
The reason we need a `filters` group rather than overriding top-level vars (like in the `Base` class) is because data classes are strict on class vars/instance vars.

Reply:
Can we derive a base class like `JSONFilterableObject` and place the `filters` group and the meta class in it so others class derived from `JSONObject` won't be affected by the `filters`?

Reply:
FWIW this implementation parallels [FilterableMetaclass](https://github.com/linode/linode_api4-python/blob/dev/linode_api4/objects/filtering.py#L245] but I definitely wouldn't be opposed to splitting it up for this one :+1:

Reply:
Just realized that most our API objects are filterable so I guess we can just keep the `filters` in this class 👍